src/utils/dispute-utils.js

```diff
@@ -473,7 +484,7 @@ function getRoundPhasesAndTime(courtConfig, round, currentPhase) {
     }
 
     // If round not appealed
-    if (!currentPhase.appealed) {
```

Comment:
Note that is actually not necessary to tell if the `currentPhase` was appealed or not since we have that information directly from the `round`.  For this reason we removed the `appealed` attribute at `getAdjudicationPhase`

E.g See https://github.com/aragon/court-dashboard/pull/338/files#diff-0311c4b36f556c7a49619bd3afa3d955L341